Dockerfile

```diff
@@ -0,0 +1,22 @@
+# build the app in a separate build container
+FROM node:lts-slim as build
+# Permission for app
+RUN mkdir /app && chown node:node /app
+WORKDIR /app
+COPY --chown=node:node . .
+ENV PATH="./node_modules/.bin:$PATH"
+USER node
+RUN npm ci --no-optional && npm cache clean --force && npm run build
```

Comment:
Out of curiosity: Why do you run an npm cache clean, if the image is thrown away anyways (since it is the build image)?

Reply:
No real background - it was just copy & paste from an existing Dockerfile. Maybe this was even from pre-container era ;)